core/capabilities/ccip/ccip_integration_tests/ccipreader/ccipreader_test.go

```diff
@@ -254,6 +255,13 @@ func TestCCIPReader_MsgsBetweenSeqNums(t *testing.T) {
 								consts.EventAttributeSequenceNumber: {Name: "message.header.sequenceNumber"},
 							},
 						},
+						OutputModifications: codec.ModifiersConfig{
+							&codec.WrapperModifierConfig{Fields: map[string]string{
+								"Message.FeeTokenAmount":      "Int",
+								"Message.FeeValueJuels":       "Int",
```

Comment:
Why did FeeValueJuels and FeeTokenAmount needed to be added here if they were already been specified as ints below (on line 277 and 278 below) and the tests were passing?

Reply:
Are you referring to these lines passed into `EmitCCIPMessageSent`?
`FeeTokenAmount: big.NewInt(1),
		FeeValueJuels:  big.NewInt(2),
		TokenAmounts:   make([]ccip_reader_tester.InternalEVM2AnyTokenTransfer, 0),`
If so, this is a direct call to geth contract wrapper that emits ccipMessageSent event, which accepts the onchain equivalent uint256 which in GO through the ABI maps to big.Int.

The ccip offchain structure or i.e. the chain agnostic structure that we want to parse the results into has these values wrapped inside of this struct, so with the codec modifier we are doing the same.
`type BigInt struct {
	*big.Int
}`

The tests were passing because initially the unit test allowed these values to be zero and afterwards was changed so that a helper struct was passed into QueryKey() which did not have big int wrapped. You can see this in the ccip repo changes linked to this PR.

Reply:
Ok yea sounds good. That answers my question.